Tests/designspaceLib/designspace_test.py

```diff
@@ -785,3 +791,18 @@ def test_documentLib(tmpdir):
     assert dummyKey in new.lib
     assert new.lib[dummyKey] == dummyData
 
+@pytest.mark.skipif(pathlib is None, reason="pathlib not installed")
+def test_read_with_path_object():
+    import pathlib
+    source = (pathlib.Path(__file__) / "../data/test.designspace").resolve()
+    assert source.exists()
+    doc = DesignSpaceDocument()
+    doc.read(source)
+
+@pytest.mark.skipif(pathlib is None, reason="pathlib not installed")
+def test_with_with_path_object(tmpdir):
+    import pathlib
+    dest = pathlib.Path(tmpdir) / "test.designspace"
```

Comment:
Do `pathlib.Path(str(tmpdir))` here
The tmpdir fixture returns a py.path.local object (yet another path library)

Reply:
Or use `tmp_path` instead of `tmpdir` in pytest 3.9+.

Reply:
Does pytest now requires pathlib2 backport on py27?

Reply:
Yes if you use tmp_path.

Reply:
Whatever works. I’m ok with requiring the latest pytest to run tests (which is what the CI runs anyway).